src/dynamics/ccd/mod.rs

```diff
@@ -216,6 +216,11 @@
 //! For shapes that are intended to be solid from the inside, it is recommended
 //! to use convex decomposition instead.
 //!
+//! If you must use triangle mesh colliders and are having stability issues, consider
```

Comment:
Is this only relevant for CCD?

Reply:
This section in the CCD docs is under an "Other Ways to Avoid Tunneling" heading. But I'll mention more explicitly in the `CollisionMargin` docs that it can be especially useful for trimesh colliders

Reply:
I'd add it to the trimesh constructor itself, since otherwise a user wouldn't stumble upon the fix without knowing about it already.